index.js

```diff
@@ -79,6 +79,7 @@ module.exports = {
         argsIgnorePattern: "^_",
       },
     ],
+    "@typescript-eslint/prefer-ts-expect-error": "error",
```

Comment:
Something to look into in the future: `plugin:@typescript-eslint/strict` extends this rule. Since we already extend `/recommended` and `/recommended-requiring-type-checking` we may be able to unlock a new set of rules that allow us as a team to write more uniformly similar code: strict rules have a lock icon associated with them https://typescript-eslint.io/rules/.

Examples of "incorrect code" I've seen in PRs, whether commented on or not, that some of these rules would enforce consistently across the team at commit-time: https://typescript-eslint.io/rules/no-unnecessary-boolean-literal-compare/, https://typescript-eslint.io/rules/no-unnecessary-condition/, https://typescript-eslint.io/rules/no-non-null-asserted-nullish-coalescing/, https://typescript-eslint.io/rules/no-confusing-non-null-assertion/